instrumentation/opentelemetry-instrumentation-pymemcache/tests/test_pymemcache.py

```diff
@@ -33,6 +35,14 @@
 TEST_HOST = "localhost"
 TEST_PORT = 117711
 
+pymemcache_version = get_package_version.parse(pymemcache_package_version)
+pymemcache_version_lt_2 = pymemcache_version < get_package_version.parse(
+    "2.0.0"
+)
+pymemcache_version_gt_341 = pymemcache_version > get_package_version.parse(
```

Comment:
3.4.2 removed spaces in the return of the stats command.

Reply:
Curious is if this is really necessary to have a separate version test just simply for the existence/non-existence of trailing whitespace after commands that take no arguments. It does not seem that significant IMO in terms of a telemetry standpoint. Should we just modify the below test for `stats`?

Reply:
@lzchen, I went back and forth on that. At the end of it, I felt like a breaking change in a non major version bump was a pretty big change given the nature of how the test is working.

If you or others have strong opinions, I am all ears.

Reply:
Not a strong opinion.